### ISSUE TITLE
FUGU14 Added Support for A12 Devices from iOS 14.2

### ISSUE THREAD

**Issue by justaplane** [pwn20wndstuff/Undecimus#2278]
Don't have twitter, otherwise I would tweet this.

The documentation for FUGU14 states that:

"Fugu14 should support all arm64e devices (iPhone XS and newer) on iOS 14.3 - 14.5.1.
**Support for lower versions (down to 14.2) can be added by editing arm/shared/ClosurePwn/Sources/ClosurePwn/PwnClosure.swift and arm/shared/KernelExploit/Sources/KernelExploit/offsets.swift."**

This would allow devices such as the iPad Mini (5th generation) - A12 processor to have a persistent jailbreak on an IOS version back to 14.2.

Realizing that 14.4 -14.5.1 is currently supported through AltStore UnC0ver, wondering if this can be updated per above, and / or whether a standalone "side load" option could be supported for folks who have their own developer accounts and can sign themselves.

**Comment by sadohat** [comment]
No, the untethered jailbreak only goes from 14.3 to 14.5.1. 14.2 support can be added by switching to another exploit. However I did hear that fugu14 might support 14.0 to 14.5.1 in the near future, and on the same repo it says you can apply the jailbreak to checkra1n.
